### DISCUSSION TITLE
Virtual Machines have same IP addresses

### DISCUSSION THREAD

**Discussion by deanmagara** [utmapp/UTM#5547] [Linux]
Hello there, 
I am fairly new to the virtual machine field. I downloaded ubuntu and kali linux. All the network modes font work except for the Emulated VLAN. but this makes the virtual machines have the IP addresses. 
How do I make them get different addresses? I read somewhere to Bridged network mode and modify it to my specifications but as I said I am fairly new, so I don't know what am doing?
Thanks in advance.

**Comment by KevinMGranger** [comment]
Did you clone the VMs? I did and had the same issue. This [Server Fault answer](https://serverfault.com/questions/1114811/cloning-a-utm-virtual-machine-ip-address-the-same/1115652#1115652) worked for me:

> Turn off the vm and go to its setting>network click on Random in the Mac address, switch on the vm you'll see it should have a new IP

(be sure to upvote them if it worked for you!)

**Comment by IngoMeyer441** [comment]
I have the same problem as well and try to automate the cloning process from the command line using `utmctl`. `utmctl clone` works, but is it possible to edit the VM settings (like the MAC address) from the command line?

**Reply by IngoMeyer441** [reply]
I found an answer myself: <https://serverfault.com/a/1162322/582597>.
